src/embind/embind_gen.js

```diff
@@ -454,14 +458,27 @@ var LibraryEmbind = {
 
     print() {
       const out = ['{\n'];
+      const publicSymbols = [];
       for (const def of this.definitions) {
+        if (def.hasPublicSymbol) {
+          publicSymbols.push(def.name);
+        }
         if (!def.printJs) {
           continue;
         }
         def.printJs(out);
       }
-      out.push('}')
-      console.log(out.join(''));
+      out.push('}\n');
+      let updateExports = '() => {\n';
+      for (const publicSymbol of publicSymbols) {
+        updateExports += `__exp_${publicSymbol} = Module['${publicSymbol}'];\n`
```

Comment:
So these extra assignment happen during `onInit` in the final output module?  They don't happen at AOT time, right?  Or do they happen both at AOT time and at runtime?

Reply:
They happen just in the final output at runtime.